rspec-tools/rspec_tools/validation/metadata.py

```diff
@@ -20,6 +92,37 @@ def validate_metadata(rule_language: LanguageSpecificRule):
   validate_status(rule_language)
   validate_security_standards(rule_language)
 
+
+def validate_rule_metadata(rule: GenericRule):
+  '''In addition to the test carried out by validate_metadata for each language specification,
+     a rule must have at least one language (unless it is part of the list of exception).
+  '''
+  specializations = list(rule.specializations)
+  if rule.id in RULES_WITH_NO_LANGUAGES:
+    if specializations:
+      # When this triggers, ask yourself whether the rule should be removed from RULES_WITH_NO_LANGUAGES
+      raise RuleValidationError(f'Rule {rule.id} should have no specializations')
```

Comment:
Perhaps add the (shorter version of) the comment to the error message?
e.g.
```suggestion
      # When this triggers, ask yourself whether the rule should be removed from RULES_WITH_NO_LANGUAGES
      raise RuleValidationError(f'Rule {rule.id} should have no specializations. Forgot to remove it from the list?')
```

Reply:
Done in d8ab57a.